.github/workflows/rust.yml

```diff
@@ -75,7 +75,9 @@ jobs:
       - name: Checkout Crate
         uses: actions/checkout@v4
       - name: Checkout Toolchain
-        uses: dtolnay/rust-toolchain@nightly
+        uses: dtolnay/rust-toolchain@master
```

Comment:
Ironically this unpins other part of the CI: the action. If there's an unreleased feature needed (didn't check) then it'd be better to just use the commit hash.

Reply:
I *believe* the latest nightlies come with master. But let me try pinning to some speciifc commit and see if it works.

Reply:
Ok, pinned to the `v1` tag which should be fine.